File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/fragment/entitieslist/core/taskslist/TasksListFragment.java
package com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.core.taskslist;

import com.github.valhallalabs.laverna.persistent.entity.Task;
import com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.EntitiesListWithSearchFragment;

/**
 * @author Vadim Boitsov <vadimboitsov1@gmail.com>
 */

public interface TasksListFragment extends EntitiesListWithSearchFragment {

    void openRelatedNote(Task task);

}

File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/fragment/entitieslist/core/taskslist/TasksListFragment.kt
package com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.core.taskslist

import com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.EntitiesListWithSearchFragment
import com.github.valhallalabs.laverna.persistent.entity.Task

/**
 * @author Vadim Boitsov <vadimboitsov1></vadimboitsov1>@gmail.com>
 */
interface TasksListFragment : EntitiesListWithSearchFragment {
    fun openRelatedNote(task: Task?)
}
